vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -49,7 +49,14 @@ import (
 	resourceclient "k8s.io/metrics/pkg/client/clientset/versioned/typed/metrics/v1beta1"
 )
 
-const defaultResyncPeriod time.Duration = 10 * time.Minute
+const (
+	betweenScaleSubresourceRefreshes time.Duration = 10 * time.Minute
+	containerCacheCleanupLoopPeriod  time.Duration = time.Hour
+	containerCacheRefreshLoopPeriod  time.Duration = time.Minute
```

Comment:
Seeing how these two are used I'd say they need renaming.

And seeing that I got some of these consts confused, maybe they could be commented to explain their meaning.

Reply:
Done.